pom.xml

```diff
@@ -171,7 +172,7 @@
       <name>jcenter</name>
       <url>https://jcenter.bintray.com</url>
       <snapshots>
-        <enabled>false</enabled>
+        <enabled>true</enabled>
```

Comment:
We don't want to get snapshots from remote repos. If you need a snapshot version of WebAnno, best check it out locally.